cycles-ledger/src/storage.rs

```diff
@@ -1725,17 +1799,22 @@ pub async fn withdraw(
 
 pub async fn create_canister(
     from: Account,
+    spender: Option<Account>,
     amount: u128,
     now: u64,
     created_at_time: Option<u64>,
     argument: Option<CmcCreateCanisterArgs>,
-) -> Result<CreateCanisterSuccess, CreateCanisterError> {
-    use CreateCanisterError::*;
+) -> Result<CreateCanisterSuccess, CreateCanisterFromError> {
+    use CreateCanisterFromError::*;
+
+    if amount == 0 {
```

Comment:
Again, I'm confused about this check. If `amount = 1`, it wouldn't work either, right?

Reply:
Error case removed after discussing it on Slack